### ISSUE TITLE
AI generation and export not properly working on macOS & better error handling

### ISSUE THREAD

**Issue by dantaeusb** [bolphen/rimworld-avatar#1]
I am sure have an executable script that runs Stable Diffusion and perfectly capable of creating avatars.

When I try to run a prompt, I see an "AI gen process started" message, window disappears, no errors in logs, however, nothing happens – the script is not running properly.

If I deliberately use the wrong path, the same "AI gen process started" message appears, and the window stays in place. Yet, no indication that things went wrong aside from logs.

Manually, I can run something like `/Users/dantaeusb/Projects/Personal/stable-diffusion/rimworld-repaint.py "/Users/dantaeusb/Library/Application Support/ludeon.rimworld/avatar/Ash_Watkins_649.png"` and get a result in-place after.

I am assuming that either it exits for some reason in when called from RimWorld, or the reason might be incorrect usage of `Application.persistentDataPath` – I see the mod uses `Application Support/ludeon.rimworld` folder for storage, like Unity's default, while RimWorld itself stores data in `Application Support/RimWorld`. Perhaps something else needs to be used instead?

<img width="597" alt="Screenshot 2024-10-24 at 23 57 43" src="https://github.com/user-attachments/assets/26df42ac-1e98-4518-937e-9f11d55efcd1">

I will try to continue my own investigation of that, but I haven't worked with RimWorld mods, so any help is appreciated.

**Comment by bolphen** [comment]
You are right that there is no error handling whatsoever. I just pushed a new version that dumps the error message so hopefully it'd make debugging a little bit easier.
